code-style/git/README.md

```diff
@@ -1,6 +1,8 @@
 Git
 ===
```

Comment:
This is the wrong heading style, see our heading guidelines!

Reply:
Great add, would you mind changing this heading while you're in the file?

Reply:
Yuppers! I'll get this